packages/core/upload/admin/src/pages/App/components/Header.tsx

```diff
@@ -36,9 +45,9 @@ export const Header = ({
       })}
       subtitle={
         breadcrumbs &&
+        typeof breadcrumbs !== 'boolean' &&
```

Comment:
why not set breadcrummbs to null by default? instead of having an Array<> | boolean ?

Reply:
I agree, I saw this in the other Header component and I don't understand why boolean is an accepted type. I think you could just do `Boolean(breadcrumbs)` instead of this check whether boolean is an accepted type or not.

Reply:
Basically it is because of the way the value is passed in the MediaLibraryComponent https://github.com/strapi/strapi/blob/129a1419ddf02d6bc81f30fb03e2264a1eb5ec0f/packages/core/upload/admin/src/pages/App/MediaLibrary/MediaLibrary.tsx#L242 for sure instead of being a && it could be a ternary where we send a null value if the isCurrentFolderLoading is false

Reply:
Yes, let's do that. I find it so confusing that breadcrumbs is either an array or false.

Reply:
I changed the way we pass the breadcrumbs prop in the MediaLibrary component